db/migration/cmd/main.go

```diff
@@ -27,14 +27,20 @@ func main() {
 	sslmode := viper.GetString("dbsslmode")
 
 	var forceVersion int
+	var direction string
 	var err error
-	if len(os.Args) > 1 {
-		forceVersion, err = strconv.Atoi(os.Args[1])
+	if len(os.Args) > 2 {
+		direction = os.Args[1]
+		forceVersion, err = strconv.Atoi(os.Args[2])
 		if err != nil {
 			log.Fatalf("ERROR: Could not convert force version from string to int")
 		}
+	} else if len(os.Args) > 1 {
+		direction = os.Args[1]
+		forceVersion = -1
```

Comment:
It might have been because of the issues you mentioned above.. it seems like I forgot to push up some changes that I made! I think I fixed everything though and it seems to be running correctly for me!

Reply:
Hmm it's still not working for me. I think the issue is that if I run the command with just `force_version` (for example `make migrate_database force_version=22`) and don't include `direction` then there's only one argument and that's getting set to the `direction` variable here. So I think we need some way to check the value and make sure it's getting set to the right variable? Unless I'm missing something?